packages/backend.ai-ui/src/components/baiClient/FileExplorer/ControlItems.tsx

```diff
@@ -0,0 +1,154 @@
+import { ControlItemsFragment$key } from '../../../__generated__/ControlItemsFragment.graphql';
+import { useMergedAllowedStorageHostPermission } from '../../../hooks';
+import { BAITrashBinIcon } from '../../../icons';
+import BAIFlex from '../../BAIFlex';
+import useConnectedBAIClient from '../../provider/BAIClientProvider/hooks/useConnectedBAIClient';
+import { VFolderFile } from '../../provider/BAIClientProvider/types';
+import { FolderInfoContext } from './BAIFileExplorer';
+import { useMutation } from '@tanstack/react-query';
+import { App, Button, theme } from 'antd';
+import _ from 'lodash';
+import { DownloadIcon } from 'lucide-react';
+import { use } from 'react';
+import { useTranslation } from 'react-i18next';
+import { graphql, useFragment } from 'react-relay';
+
+interface ControlItemsProps {
+  ControlItemsFrgmt?: ControlItemsFragment$key | null;
```

Comment:
The variable name should indicate its type, so it is clear whether it is a `VirtualFolder`, `ComputeSession`, etc. Please improve the variable name accordingly. Also, props should never start with an uppercase letter.